docs/tutorials/troubleshoot-a-course.md

```diff
@@ -59,9 +59,12 @@ Leverage the `static-docker-commands` and `static-file-ops` to reduce post submi
 
 ## Unexpected Grading Results
 
-When graders work locally but unexpectedly during docker grading,
+When graders work locally but unexpectedly give the wrong output during docker grading,
 the easiest way to pinpoint issues is to manually step through the grading process inside the grading container.
 
+The assignment grader image will likely follow the naming scheme of `autograder.<course-name>.<assignment-name>`.
```

Comment:
s/likely/generally/

It could be just me, but to me "likely" implied some sort of probabilistic process while "generally" implies a rule with exceptions (a deterministic process).